spec/abstractops.html

```diff
@@ -1508,7 +1508,7 @@ <h1>PrepareTemporalFields ( _fields_, _fieldNames_, _requiredFields_ )</h1>
     <emu-note>The value of ? ToIntegerOrInfinity(*undefined*) is 0.</emu-note>
     <emu-alg>
       1. Assert: Type(_fields_) is Object.
-      1. Let _result_ be ? OrdinaryObjectCreate(%Object.prototype%).
+      1. Let _result_ be ! OrdinaryObjectCreate(%Object.prototype%).
```

Comment:
Though this isn't an options object, do we still want it doing lookups on Object.prototype? Does it get passed back to user code and that's why?

Reply:
`Object.create(null)` objects behave weirdly, and I think are to be avoided in general. I agreed with @bakkot's suggestion here because ECMA-402 has [GetOptionsObject](https://tc39.es/ecma402/#sec-getoptionsobject) which does `Object.create(null)` if you pass it `undefined`, and options objects should be consistent throughout JS.

However, I don't think we need that consistency with every object that's created internally within Temporal, and there are instances elsewhere in ECMA-262 where normal objects are created.

(For example, changing this to `null` would mean that something like `plainDate.getFields().hasOwnProperty('year')` would throw, because `hasOwnProperty` isn't looked up on Object.prototype)